synapse/res/templates/style.css

```diff
@@ -0,0 +1,30 @@
+html {
+    height: 100%;
+}
+
+body {
+    background: #f9fafb;
+    max-width: 680px;
+    margin: auto;
+    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+}
+
+.mx_Header {
+    border-bottom: 3px solid #ddd;
+    margin-bottom: 1rem;
+    padding-top: 1rem;
+    padding-bottom: 1rem;
+    text-align: center;
+}
+
+@media screen and (max-width: 1120px) {
+    body {
+        font-size: 20px;
+    }
+
+    h1 { font-size: 1rem; }
+    h2 { font-size: .9rem; }
+    h3 { font-size: .85rem; }
+    h4 { font-size: .8rem; }
+
+}
```

Comment:
Why do we want to only apply these styles to screens < 1120px wide, when the body has a max-width of 680px anyway?

Reply:
@gsouquet Any thoughts here? Not sure where this came from.

Reply:
I carried that over from the existing CSS. 

This is probably to optimise readibility on different device types... probably on mobile someone judged that text was looking too big.
Keeping it or removing it should not make a massive difference. We will likely iterate over that in the future